tests/test_session_0.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+
+# Author: Leonardo Gama (@leogama)
+# Copyright (c) 2008-2016 California Institute of Technology.
```

Comment:
nit: only need current year in copyright

Reply:
Oops! I just copied it and didn't see. I usually set my own copyright notices in Vim and it puts the correct year for me... 😓

Reply:
Should I put just this?
```
# Copyright (c) 2022 The Uncertainty Quantification Foundation.
```

Reply:
yep